### PR TITLE
Merge Inorder_Traversal.cpp

### PR DESCRIPTION
**Pull request by asciarp08**

### CODE REVIEW

File: Hashing/Fraction_Interviewbit.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            res.push_back(')');
+            break;
+        }
+        
+        m[rem] = res.size();
+        rem *= 10;
+        res.push_back('0' + rem/d);
+    }
+    return res;
+}
```
Comment (vanshikaarora) [inline Hashing/Fraction_Interviewbit.cpp]: Please add headers or you can choose to remove this file

File: Hashing/Window_String_Interviewbit.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            {
+                res.clear();
+                res.append(S.begin()+p, S.begin()+q+1);
+                minLength = currLength;
+            }
+        }
+    }
+    
+    return res;
+}
```
Comment (vanshikaarora) [inline Hashing/Window_String_Interviewbit.cpp]: Please add headers or you can choose to remove this file

File: Hashing/Fraction_Interviewbit.cpp (hunk 1 above)
Comment (asciarp08) [reply Hashing/Fraction_Interviewbit.cpp]: Removed

File: Hashing/Window_String_Interviewbit.cpp (hunk 2 above)
Comment (asciarp08) [reply Hashing/Window_String_Interviewbit.cpp]: Removed

**Pull request merged**